reproduciblebuilds/README.md

```diff
@@ -0,0 +1,69 @@
+# Bootstrappable Cryptopower Builds
+
+This directory contains the files necessary to perform bootstrappable cryptopower builds.
+
+Bootstrappability furthers our binary security guarantees by allowing us to audit and reproduce our toolchain instead of blindly trusting binary downloads.
+
+We achieve bootstrappability by using Docker.
+
+## Usage
+
+Note: You need to have [Docker](https://www.docker.com/) to build.
+
+in the cryptopower root directory and execute:
+
+`./reproducible_builds.sh {target}`
+
+select your target from [Build Targets](https://github.com/crypto-power/cryptopower/tree/master/reproduciblebuilds#build-targets)
+
+e.g `./reproducible_builds.sh linux-amd64-binary`
+
+the reproduced binary can be found in this directory
+
+### Verification
+
+You can then verify if the binaries are reproducible using a tool like [diffoscope](https://diffoscope.org/) or [reprotest](https://salsa.debian.org/reproducible-builds/reprotest)
+
+#### Using Diffoscope
+
+e.g `diffoscope binary1 binary2`
+
+The 2 binaries should be the official cryptopower release binary which can be found on the [release page](https://github.com/crypto-power/cryptopower/releases) and the binary you reproduced which can be found in this directory
```

Comment:
I think here you meant to discuss about the official reference binary since the other one is the user generated from his own source code.

Reply:
that's what the comment says

Reply:
>The 2 binaries should be the official cryptopower release binary which can be found on the ...

unless I am getting it wrong, the statement above implies the opposite of when you just said.

Reply:
part one of the statement

> The 2 binaries should be the official cryptopower release binary which can be found on the [release page](https://github.com/crypto-power/cryptopower/releases)

part 2 of the statement

> and the binary you reproduced which can be found in this directory